lib/pacemaker/pcmk_sched_migration.c

```diff
@@ -277,11 +281,11 @@ pcmk__order_migration_equivalents(pe__ordering_t *order,
     then_task = task_from_action_or_key(order->rh_action,
                                         order->rh_action_task);
     if ((first_task == NULL) || (then_task == NULL)) {
-        goto cleanup_order;
```

Comment:
Would it be worth logging here just in case this ever actually happens?

Reply:
We can CRM_CHECK() it

Reply:
Actually we can skip the check altogether, it won't hit any of the following conditions and will end up at cleanup_order

Reply:
Updated commit